im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,425 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Im Gespräch - Hefte der Martin Buber-Gesellschaft (German)</title>
+    <title-short>Im Gespräch</title-short>
+    <id>http://www.zotero.org/styles/im-gespraech-buber-gesellschaft</id>
```

Comment:
We simply drop diacritics for the style file name and ID, and rarely use (parts of) subtitles. I much rather see `im-gesprach.csl` than `im-gespraech-buber-gesellschaft.csl`.